tests/unit/command/test_repro.py

```diff
@@ -15,14 +15,14 @@
 }
 
 
-def test_default_arguments(dvc_repo, mocker):
+def test_default_arguments(mocker):
```

Comment:
Same as in https://github.com/iterative/dvc/pull/3139/files , these `dvc_repo` fixtures needed to ensure that we are not accidenatlly running in the core repo.